ingestion/ingestion_server/ingestion_file_upload_rest_api.py

```diff
@@ -657,17 +672,17 @@ def delete_all():
     try:
         for each_file in files_list:
             if 'type' not in each_file.metadata:
-                success, key_message = delete_key(each_file)
+                success, key_message = delete_key(each_file.metadata['asset_uuid'])
```

Comment:
As far as I understand this was taken care of in a previous PR - make sure there are no conflicts with master branch